usaspending_api/references/migrations/0054_auto_20170308_2100.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-03-08 21:00
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('references', '0053_auto_20170210_1814'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='subtieragency',
+            name='abbreviation',
+            field=models.CharField(blank=True, max_length=150, null=True, verbose_name='Agency Abbreviation'),
```

Comment:
This might be a dumb question - but above we have Agency Abbreviation and Subtier Abbreviation, do we need to use both here too? Or does Agency Abbreviation pull for both?